content/academy/become_an_apify_expert/solutions/managing_source.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Managing source code quiz answers
+description: View in-depth answers for all three of the quiz questions that were provided in the corresponding lesson about managing source code.
+menuWeight: 3
+paths:
+    - become-an-apify-expert/solutions/managing-source
+---
+
+## [](#quiz-answers) Quiz answers
```

Comment:
Could you make this heading consistent with the meta title? Currently, the file name, meta title, and H1 are all different.

Also, this should be H1, no?

Reply:
Also, if you're ok with making the questions bold in the other file, please also update here.